modes/fr/contact.md

```
@@ -0,0 +1,55 @@
+
+2. **Classifier le type de contact** — demander au candidat ou inférer du contexte :
+   - **Recruteur** — personne dont le rôle est talent acquisition, sourcing ou recruiting
+   - **Hiring Manager** — la personne qui dirige l'équipe qui recrute
+   - **Pair** — quelqu'un avec un rôle similaire dans l'équipe (referral indirect)
+   - **Intervieweur** — quelqu'un qui va t'interviewer (date connue)
+
+3. **Sélectionner la cible primaire** : la personne qui bénéficierait le plus de ta présence dans l'équipe
+
+4. **Générer le message** avec un framework de 3 phrases adapté au type de contact :
+
+   ### Recruteur
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)